common/src/main/res/values/strings.xml

```diff
@@ -795,6 +795,7 @@
     <string name="show_share_logs">Show and share logs</string>
     <string name="show_entity_title">Show entity name</string>
     <string name="show_unit_title">Show entity unit</string>
+    <string name="forward_taps_title">Forward taps to entity</string>
```

Comment:
Reading this, I was very confused about what the option will actually do. While it may be because I'm not a native English speaker, that will mean some other people probably also won't understand it.

Why not mimic what the entity state widget on the phone does (which is much more obvious in my opinion) and make it tap action: toggle or tap action: refresh?

Reply:
Hello,

Thanks for the feedback! I did think quite a bit about the wording and was left unsatisfied too :D

1) the word "toggle" seemed a bit too restrictive to me: indeed, depending on the entity domain, it can execute the following services:
- toggle
- lock  / unlock
- arm / disarm alarm
- press (for a `button` / `input_button`)
- turn on (for a `scene`)

Even if we take "toggle" as a loose term used to convey flipping between two states, it does not include the last two cases where the action is unary.

That being said, maybe given the precedent set by the widget (that I was unaware of), and the fact that it might be easier for users to understand (even though technically less accurate), we should go for it?

2) it might seem to the user that they are choosing between two exclusive options: refresh OR toggle. In actuality, both would refresh the complication - with one additionally triggering the entity prior to the refresh.

3) material3 / compose for wear do not appear to have a way to create drop-down dialogs easily.

Proposal: how about keeping this toggle, and changing the text to "Toggle entity" to make it more user-friendly and consistent with the widget?

Reply:
Naming things is often harder than building them :D

> the word "toggle" seemed a bit too restrictive to me

While I can understand your points, your two "That being said" points feel more important to me than being technically correct.

 - consistency with the widget/other parts of the app for what is the same feature, instead of giving different names for the same feature
 - when you don't know Home Assistant inside out, it is easier to understand what will happen when you "toggle a scene/toggle a button" vs "forward tap to a scene/forward tap a button", even if it is technically not toggling

> it might seem to the user that they are choosing between two exclusive options: refresh OR toggle

I'd still expect feedback when I tap something which in addition to doing the thing, would be refreshing to show the updated state, so it's implied?

> keeping this toggle, and changing the text to "Toggle entity when tapped"

The issue with keeping this as a toggle is that it is unclear what happens when you uncheck it - there is no clear opposite. The native pattern for dropdowns on a watch would be 2 chips/buttons that are radio buttons.

Reply:
There is a clear opposite to "toggle entity when tapped": "don't toggle entity when tapped" :) IMO that's quite intuitive given the usual semantics of toggles.

I defer to your opinion in this matter - but my 2 cents is that "toggle" is not ideal. On top of the cases I listed above, consider the case of a sensor entity - how are users expected to understand "toggle" in this context? It sounds like it would enabling / disabling the sensor, doesn't it? In any case, this is outside the scope of this PR since it would require refactoring the other uses of the "toggle" word in similar context (eg. the widget). Just wanted to give my opinion :)

Will work on the radio button later. Thanks for the discussion!